var/spack/repos/builtin/packages/py-termcolor/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTermcolor(PythonPackage):
```

Comment:
Hipster stuff.

Reply:
Will keep this as I see this as a dependency in tensorflow.

Reply:
Clear. Seems though that Jenkins can't parse the XML due to escape characters. Just tested it locally, and it does not seem to sense "dumb" terminals. Just always prints control characters?

Reply:
Here's how it's done right: https://github.com/erikrose/blessings

**This package actually has some terminal-sensing logic.**

Reply:
@matz-e : do you want me to change anything here now? Otherwise I intend merge this now.